src/drive/web/modules/viewer/FilesViewer.jsx

```diff
@@ -115,20 +130,40 @@ const FilesViewer = ({ filesQuery, files, fileId, onClose, onChange }) => {
     }
   }, []) // eslint-disable-line react-hooks/exhaustive-deps
 
+  useEffect(() => {
+    const getDecyptedURLIfNecessary = async () => {
```

Comment:
```suggestion
    const getDecryptedURLIfNecessary = async () => {
```

Reply:
Looks like you have better eyes than me :)